agent/php_txn.c

```diff
@@ -1109,3 +1109,101 @@ nr_status_t nr_php_txn_end(int ignoretxn, int in_post_deactivate TSRMLS_DC) {
 
   return NR_SUCCESS;
 }
+
+extern void nr_php_txn_add_code_level_metrics(
+    nr_attributes_t* attributes,
+    const nr_php_execute_metadata_t* metadata) {
+/* Current CLM functionality only works with PHP 7+ */
+#if ZEND_MODULE_API_NO < ZEND_7_0_X_API_NO /* PHP < PHP7 */
+  (void)attributes;
+  (void)metadata;
+  return;
+}
+#else
```

Comment:
Why not make whole function available only for PHP 7+? See [this comment](https://github.com/newrelic/newrelic-php-agent/pull/585/files?diff=split&w=1#r1043702320).

Reply:
Addressed in 3e7ec823504a22e68c094b947fd86af06ebd41d9.